server/rest/src/test/java/org/infinispan/rest/resources/CacheV2ResourceTest.java

```diff
@@ -71,8 +71,9 @@ protected void defineCaches(EmbeddedCacheManager cm) {
    @Override
    public Object[] factory() {
       return new Object[]{
-            new CacheV2ResourceTest().withSecurity(true),
-            new CacheV2ResourceTest().withSecurity(false)
+            new CacheV2ResourceTest().withSecurity(false),
```

Comment:
same comment as above

Reply:
I will probably add SSL + HTTP1 to either cachev1 or cachev2, to avoiding passing the 1.000 tests mark :)